CHANGELOG.md

```diff
@@ -1,3 +1,64 @@
+  * **roads**: Drop `all_networks` and `all_shield_texts` properties from roads at low- and mid-zooms. [Issue #1642](https://github.com/tilezen/vector-datasource/issues/1642).
+  * **roads**: Document new heavy good vehicle (hgv, or truck) properties in schema (but not yet added to tile content), thanks @musculman! [Issue #1553](https://github.com/tilezen/vector-datasource/issues/1553).
+  * **traffic_flow**: Add new optional layer definition, thanks @conor-ettinoffe-here! [PR #1705](https://github.com/tilezen/vector-datasource/pull/1705).
+  * **traffic_incidents**: Add new optional layer definition, thanks @conor-ettinoffe-here! [PR #1705](https://github.com/tilezen/vector-datasource/pull/1705) and [PR #1719](https://github.com/tilezen/vector-datasource/pull/1719).
+  * **water**: Merge water lines, thanks @sensescape. [Issue #1135](https://github.com/tilezen/vector-datasource/issues/1135).
+  * **water**: Remove water point labels generated from lines. [Issue #1702](https://github.com/tilezen/vector-datasource/issues/1702).
+  * **water**: Simplify at zoom 8 to match the transition from Natural Earth to OpenStreetMap, significantly reducing file size at that zoom. [Issue #1477](https://github.com/tilezen/vector-datasource/issues/1477) and [PR #1714](https://github.com/tilezen/vector-datasource/pull/1714).
+  * **water**: Drop smaller water features across at all zooms but max. [Issue #1477](https://github.com/tilezen/vector-datasource/issues/1477) and [PR #1714](https://github.com/tilezen/vector-datasource/pull/1714).
+  * **water**: Drop name that don't fit on feature at all zooms but max. [Issue #1477](https://github.com/tilezen/vector-datasource/issues/1477) and [PR #1714](https://github.com/tilezen/vector-datasource/pull/1714).
+  * **water**: Truncate water and earth `min_zoom` floats to tenths place. [Issue #1477](https://github.com/tilezen/vector-datasource/issues/1477) and [PR #1714](https://github.com/tilezen/vector-datasource/pull/1714).
+
+  #### BUG FIXES
+
+  * **boundaries**: Drop intermediate buffered land polygons from low zooms. [PR #1699](https://github.com/tilezen/vector-datasource/pull/1699).
+  * **boundaries**: Fix recurring issue with martime_boundary property assignment. [Issue #1726](https://github.com/tilezen/vector-datasource/issues/1726)
+  * **pois**: Allow no-name `drinking_water` and `playground` features.
+
+  #### DOCUMENTATION CHANGES
+
+  * Updated Layers documentation for v1.6 schema changes.
+  * TODO: Updated tilejson/tilejson.json.erb for v1.5 and v1.6 schema changes.
+
+  #### INTERNAL CHANGES
+
+  * Add gunicorn to dependencies, thanks @rwrx. [PR #1690](https://github.com/tilezen/vector-datasource/pull/1690) and [PR #1691](https://github.com/tilezen/vector-datasource/pull/1691).
```

Comment:
PR #1691 relates to the use of raw strings in regular expressions (the following item), not `gunicorn`.